R/calculate-effect-sizes.R

```diff
@@ -1,3 +1,55 @@
+if(getRversion() >= "2.15.1")  utils::globalVariables(c("ES", "Est", "SE", "CI_upper", "CI_lower"))

```

Comment:
This is used to handle the warning message "no visible binding for global variable" in R CMD check.